src/main/java/sparksoniq/spark/iterator/flowr/expression/OrderByClauseExprWithIterator.java

```diff
@@ -22,36 +22,32 @@
 
 import sparksoniq.jsoniq.compiler.translator.expr.flowr.OrderByClauseExpr;
 import sparksoniq.jsoniq.runtime.iterator.RuntimeIterator;
-import sparksoniq.jsoniq.runtime.metadata.IteratorMetadata;
 
 import java.io.Serializable;
 
-public class OrderByClauseSparkIteratorExpression implements Serializable {
+public class OrderByClauseExprWithIterator implements Serializable {
```

Comment:
How about OrderByClauseAnnotatedChildIterator? (to completely drop expr)

Reply:
I like this naming, updated as such